core-bundle/src/Csrf/ContaoCsrfTokenManager.php

```diff
@@ -63,12 +63,12 @@ public function canSkipTokenValidation(Request $request, string $tokenCookieName
         ;
     }
 
-    public function getFrontendTokenValue(): string
+    public function getDefaultTokenValue(): string
     {
-        if (null === $this->frontendTokenName) {
+        if (null === $this->defaultTokenName) {
             throw new \RuntimeException('The Contao CSRF token manager was not initialized with a frontend token name.');
```

Comment:
We should adjust the exception message as well.

Reply:
good catch! fixed in 8a08cd587f6b592ee9aa95c76ec3618e36919ae8